src/Scorecard.js

```diff
@@ -0,0 +1,36 @@
+function Scorecard() {
+  this.frames = [];
+  this.score = 0;
+};
+
+Scorecard.prototype.enter = function(roll1, roll2) {
+  
```

Comment:
From user point of view, user might want to enter one roll at a time (eg, if you had a click button of the number of pins). Maybe change to enter one roll at a time?